.env

```diff
@@ -0,0 +1 @@
+DB_HOST = mongodb+srv://max:BKsqdstQahhmLK5c@cluster0.gv7ll21.mongodb.net/db-contacts?retryWrites=true&w=majority
```

Comment:
.env file must be in .gitignore. It shouldn't be visible

Reply:
added it to gitignore and created server on render.com